### ISSUE TITLE
Pipe fix attempts to change akka-streams `Sink` to fs2 `Pipe`

### ISSUE THREAD

**Issue by Daenyth** [tmciver/sink-to-pipe-scalafix#2]
Before:
```scala
import akka.stream.scaladsl.Sink
val s: Sink[Int, Future[Done]] = ???
```

After
```scala
import akka.stream.scaladsl.Sink
import fs2.Pipe
val s: Pipe[Int, Future[Done], Unit] = ???
```

**Comment by tmciver** [comment]
@Daenyth What was your expectation?  It looks like the correct transformation to me.  This scalafix is for fs2 - not for the Akka `Sink`.

**Comment by Daenyth** [comment]
It's my expectation that the fix would only edit `fs2.Sink` and not akka `Sink` (my codebase has both). After running it, the codebase wouldn't compile, because akka sink is `Sink[A, Mat]`, so the kinds don't line up - even if they had, it would be semantically wrong

**Comment by tmciver** [comment]
Oh!  I totally missed that.  I'll work on a fix.  Thanks!

**Comment by monadplus** [comment]
I'll add the test and fix it if it is still not done :)

**Comment by tmciver** [comment]
@monadplus If you're up for it, I created [a branch](https://github.com/tmciver/sink-to-pipe-scalafix/tree/akka-sink-fix) with some work towards fixing the mistaken fix to `akka.stream.scaladsl.Sink` that this issue is for.  If you run the tests you can see what's happening.  I don't know how to make scalafix only affect the `fs2.Sink` type.  I was going to try to do it this weekend, but if you'd like to take a crack at it, feel free.

**Comment by tmciver** [comment]
Resolved at 12b6784dab90941d212c61a06cf0f32e584c3474.
